bin/oref0-ed-battery.sh

```diff
@@ -2,7 +2,6 @@
 
 echo Run oref0-ed-battery
 if ! /home/edison/src/EdisonVoltage/voltage json batteryVoltage battery | grep "-66" 2>/dev/null; then
-    echo start Voltage server and trigger
-    cd /home/edison/src/EdisonVoltage && sudo ./voltage_server &
-    cd /home/edison/src/EdisonVoltage && echo 1 > /tmp/battery_trigger
+    echo start Voltage
+    sudo /home/edison/src/EdisonVoltage/voltage
```

Comment:
That's because you're not running it as root in the if condition, which you need to do for the command to update the data. If you change the if condition to use sudo, it'll get proper data back, and the whole if statement will effectively do nothing. Every time you call `voltage` it has to be as root, or it will refuse to update the data it receives.